config.py

```diff
@@ -19,6 +19,9 @@ class Config:
     CELERY_BROKER_URL = os.getenv('CELERY_BROKER_URL')
     CELERY_RESULT_BACKEND = os.getenv('CELERY_RESULT_BACKEND')
 
+    # slack bot url
+    notify_url = os.getenv('SLACK_NOTIFICATION_URL')
```

Comment:
Please maintain consistency in variable naming. As you can see, all the other config variables are uppercased.